fuel-benches/benches/set/alu.rs

```diff
@@ -0,0 +1,247 @@
+        &mut group,
+        "eq",
+        VmBench::new(Opcode::EQ(0x10, 0x11, 0x12)).with_prepare_script(vec![
+            Opcode::MOVI(0x11, 100000),
+            Opcode::MOVI(0x12, 27),
+        ]),
+    );
+
+    run_group(
+        &mut group,
+        "exp",
+        VmBench::new(Opcode::EXP(0x10, 0x11, 0x12))
+            .with_prepare_script(vec![Opcode::MOVI(0x11, 23), Opcode::MOVI(0x12, 11)]),
+    );
+
+    run_group(
+        &mut group,
+        "expi",
+        VmBench::new(Opcode::EXP(0x10, 0x11, 11))
+            .with_prepare_script(vec![Opcode::MOVI(0x11, 23)]),
+    );
+
+    run_group(
+        &mut group,
+        "gt",
+        VmBench::new(Opcode::GT(0x10, 0x11, 0x12)).with_prepare_script(vec![
+            Opcode::MOVI(0x11, 100000),
+            Opcode::MOVI(0x12, 27),
+        ]),
+    );
+
+    run_group(
+        &mut group,
+        "lt",
+        VmBench::new(Opcode::LT(0x10, 0x11, 0x12)).with_prepare_script(vec![
+            Opcode::MOVI(0x11, 100000),
+            Opcode::MOVI(0x12, 27),
+        ]),
+    );
+
+    run_group(
+        &mut group,
+        "mlog",
+        VmBench::new(Opcode::MLOG(0x10, 0x11, 0x12)).with_prepare_script(vec![
+            Opcode::MOVI(0x11, 100000),
+            Opcode::MOVI(0x12, 27),
+        ]),
+    );
+
+    run_group(
+        &mut group,
+        "mod",
+        VmBench::new(Opcode::MOD(0x10, 0x11, 0x12)).with_prepare_script(vec![
+            Opcode::MOVI(0x11, 100000),
+            Opcode::MOVI(0x12, 27),
+        ]),
+    );
+
+    run_group(
+        &mut group,
+        "modi",
+        VmBench::new(Opcode::MODI(0x10, 0x11, 27))
+            .with_prepare_script(vec![Opcode::MOVI(0x11, 100000)]),
+    );
+
+    run_group(
+        &mut group,
+        "move",
+        VmBench::new(Opcode::MOVE(0x10, 0x11))
+            .with_prepare_script(vec![Opcode::MOVI(0x11, 100000)]),
+    );
+
+    run_group(&mut group, "movi", VmBench::new(Opcode::MOVI(0x10, 27)));
+
+    run_group(
+        &mut group,
+        "mroo",
+        VmBench::new(Opcode::MROO(0x10, 0x11, 0x12)).with_prepare_script(vec![
+            Opcode::MOVI(0x11, 100000),
+            Opcode::MOVI(0x12, 27),
+        ]),
+    );
+
+    run_group(
+        &mut group,
+        "mul",
+        VmBench::new(Opcode::MUL(0x10, 0x11, 0x12)).with_prepare_script(vec![
+            Opcode::MOVI(0x11, 100000),
+            Opcode::MOVI(0x12, 27),
+        ]),
+    );
+
+    run_group(
+        &mut group,
+        "muli",
+        VmBench::new(Opcode::MULI(0x10, 0x11, 27))
+            .with_prepare_script(vec![Opcode::MOVI(0x11, 100000)]),
+    );
+
+    run_group(&mut group, "noop", VmBench::new(Opcode::NOOP));
+
+    run_group(
+        &mut group,
+        "not",
+        VmBench::new(Opcode::NOT(0x10, 0x11))
+            .with_prepare_script(vec![Opcode::MOVI(0x11, 100000)]),
+    );
+
+    run_group(
+        &mut group,
+        "or",
+        VmBench::new(Opcode::OR(0x10, 0x11, 0x12)).with_prepare_script(vec![
```

Comment:
I see this script popping up a lot, could these maybe be moved to a single constant or single expression in case it is ever updated?